cpu/native/socket_zep/socket_zep.c

```diff
@@ -575,11 +587,21 @@ static int _request_op(ieee802154_dev_t *dev, ieee802154_hal_op_t op, void *ctx)
         res = _request_transmit(dev);
         break;
     case IEEE802154_HAL_OP_SET_RX:
-        zepdev->rx = true;
+        switch (zepdev->state) {
+        case ZEPDEV_STATE_RX_RECV:
+            DEBUG("socket_zep::request_op: discard received frame\n");
```

Comment:
I'm still wondering why this happens sometimes.
Maybe during an TX -> RX transition?

Currently it's possible to receive packets in TX state, this should probably also be disallowed since a radio is not full-duplex.